sdks/python/apache_beam/ml/inference/sklearn_inference_it_test.py

```diff
@@ -23,6 +23,7 @@
 
 import pytest
 
+from apache_beam.examples.inference import sklearn_japanese_housing_regression
 from apache_beam.examples.inference import sklearn_mnist_classification
 from apache_beam.io.filesystems import FileSystems
 from apache_beam.testing.test_pipeline import TestPipeline
```

Comment:
Also, after all the imports can you add this as well?

```
# pylint: disable=wrong-import-order, wrong-import-position, ungrouped-imports, unused-import
try:
  from apache_beam.io.gcp.gcsfilesystem import GCSFileSystem
except ImportError:
  raise unittest.SkipTest('GCP dependencies are not installed')
```

Some of the tests in tox environment will run the sklearn test without the `gcp` dependencies of beam. Without `apache_beam[gcp]`, it will result in a failure. 

We have other tox envs which run sklearn tests with `apache_beam[gcp]`. That env will capture your test.